ci-operator/step-registry/vsphere/remote-worker/vsphere-remote-worker-commands.sh

```diff
@@ -8,10 +8,10 @@ trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wa
 
 remote_machineset_name="remote-worker"
 remote_worker_number="${REMOTEWORKER_NUMBER}"
-declare vsphere_connected_portgroup
+declare vsphere_bastion_portgroup
```

Comment:
other lgtm, only this is a little odd for remote worker workflow

Reply:
or we may need to separate it for remote worker

Reply:
How to introduce a new var name for remote-worker case? e.g: VSPHERE_EXTRA_LEASED_RESOURCE, once that, I think `VSPHERE_MULTIZONE_LEASED_RESOURCE` can be replaced with the the same var.  Even more, actually bastion segment is also kind of an extra segment, we also can merge it, but to be safer, maybe we can keep `VSPHERE_BASTION_LEASED_RESOURCE`.

Reply:
Both OK from me, actually I changed "VSPHERE_BASTION_LEASED_RESOURCE" to "VSPHERE_CONNECTED_LEASED_RESOURCE" in order to adapt remote worker.